### PR TITLE
[WIP] Create News Section

### PR DESCRIPTION
**Pull request by theyokohamalife**
## Part of #19

Implements the [Nuxt Headless CMS](https://content.nuxtjs.org/) to create news content for the site. This will be used to display general medical information for Japan.

### How to test

Visit `/news`

### Screenshots

Screen Shot 2021-07-14 at 10 48 59 PM

### CODE REVIEW

**Comment by renderf0x** [comment]
This is really cool, great work @theyokohamalife

File: pages/news/_slug.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<template>
```
Comment (ermish) [inline pages/news/_slug.vue:1]: is there a reason for the `_` in the name?

File: pages/news/_slug.vue [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <article class="pa-4">
+    <h1 class="text-uppercase">{{ article.title }}</h1>
```
Comment (ermish) [inline pages/news/_slug.vue:3]: nit: could just use `text-transform: uppercase;` instead of a class.

File: pages/news/_slug.vue [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <p class="grey--text text--darken-1">
+      Updated: {{ formatDate(article.updatedAt) }}
+    </p>
+    <p class="font-italic">{{ article.description }}</p>
```
Comment (ermish) [inline pages/news/_slug.vue:7]: nit: could just use `font-style: italic;`

Comment (theyokohamalife) [reply pages/news/_slug.vue:7]: You mean down in the style section, right?

File: pages/news/_slug.vue (hunk 1 above)
Comment (theyokohamalife) [reply pages/news/_slug.vue:1]: It makes it a dynamic page.
https://nuxtjs.org/docs/2.x/directory-structure/pages#dynamic-pages

**Pull request merged**